Sources/ARKit-CoreLocation/Nodes/LocationAnnotationNode.swift

```diff
@@ -59,6 +59,8 @@ open class LocationAnnotationNode: LocationNode {
 
         let distance = self.location(locationManager.bestLocationEstimate).distance(from: location)
 
+		childNodes.first?.renderingOrder = Int.max - 1000 - (Int(distance * 1000))
```

Comment:
please fix the leading whitespace here (note it's indented further than the code before and after it)

Reply:
This is a continuation of the line above it (because it exceeded the max character per line limit).  What's the proper indentation for this scenario?

Reply:
Are you sure?  This looks like an assignment (single line, non-continuation) to me.

Reply:
Yeah, you're right.  Confused with a different piece of code ☺️